docs/lorawan-on-helium/open-lns.mdx

```diff
@@ -1,15 +1,16 @@
 ---
 id: open-lns
-title: Open LNS
+title: Open LNS Overview
 pagination_label: Open LNS
 sidebar_label: Open LNS
 description: Helium Open LNS Documentation
-image: https://imgur.com/a/7jGqTV4
-slug: /lorawan-on-helium/open-lns
+image: https://docs.helium.com/img/link-image.png
+slug: /use-the-network/open-lns
```

Comment:
```suggestion
slug: /lorawan-on-helium/open-lns
```

slugs get dumb sometimes, and this is causing the build to fail when linking from the homepage.